shared/app/runtime-stats/index.tsx

```diff
@@ -74,26 +75,42 @@ const dbTypeString = (s: RPCTypes.DbType) => {
   }
 }
 
-let destroyRadar: any
+let destroyRadar: (() => void) | undefined
+let radarNode: HTMLDivElement | undefined
+const radarSize = 30
 
-const RuntimeStatsDesktop = (props: Props) => {
-  const refContainer = React.useCallback(node => {
-    if (destroyRadar) {
-      destroyRadar()
-      destroyRadar = null
-    }
-    if (node) {
-      destroyRadar = lagRadar({
-        frames: 20,
-        inset: 1,
-        parent: node,
-        size: 30,
-        speed: 0.0017 * 0.7,
-      })
-    }
-  }, [])
+const makeRadar = (show: boolean) => {
+  if (destroyRadar) {
+    destroyRadar()
+    destroyRadar = undefined
+  }
+  if (!radarNode || !show) {
+    return
+  }
+
+  destroyRadar = lagRadar({
+    frames: 5,
+    inset: 1,
+    parent: radarNode,
+    size: radarSize,
+    speed: 0.0017 * 0.7,
+  })
+}
 
-  const [hideRadar, setHideRadar] = React.useState(false)
+const RuntimeStatsDesktop = (props: Props) => {
+  const [showRadar, setShowRadar] = React.useState(flags.lagRadar)
+  const refContainer = React.useCallback(
+    node => {
+      radarNode = node
+      makeRadar(showRadar)
+    },
+    [showRadar]
+  )
+  const toggleRadar = () => {
+    const show = !showRadar
+    setShowRadar(show)
+    makeRadar(show)
+  }
```

Comment:
Any reason why this wasn't memo'ed? i.e. React.useCallback here too

Reply:
i don't think we should memo everything. this takes 3ms to render every 5 seconds